src/ripple/core/DatabaseCon.h

```diff
@@ -89,12 +89,17 @@ class DatabaseCon
         Config::StartUpType startUp = Config::NORMAL;
         bool standAlone = false;
         boost::filesystem::path dataDir;
+        static std::unique_ptr<std::vector<std::string> const> CommonPragma;
+        /// Shortcut used by the database connections that ignore the common
+        /// pragma strings
+        static const std::vector<std::string> NoCommonPragma;
     };
 
     template <std::size_t N, std::size_t M>
     DatabaseCon(
         Setup const& setup,
         std::string const& DBName,
+        bool useCommonPragma,
```

Comment:
I don't know if I like this; I'm generally allergic to functions that accept a `bool` and use it to do one or the other thing. Why can't we set the right values in `setup` directly?

Reply:
It would be pretty simple to set the values in the `Setup` object, now that you mention it. I might need to define a member function or two, just to make the internals easier, and to make it easier switch between the options for `Setup` objects that are used to create multiple `DatabaseCon`s

Reply:
PS. I went down this path because I initially thought the other ctor override would need to use the "common pragma" but without a `Setup` object being passed in. I later realized none of them do, but didn't backtrack to see if the `bool` was still a good idea. So thanks for pointing this out!